tests/PHPStan/Analyser/data/in-array.php

```diff
@@ -26,7 +28,7 @@ public function doFoo(
 			return;
 		}
 
-		if (in_array($r, $strings, true)) {
+		if (in_array($r, $moreStrings, true)) {
```

Comment:
I had to do that because `$strings` was already type-specified from previous conditionals in the same scope which messed the result up here since it's working better now.